quickstart/kubernetes/faq.md

```diff
@@ -0,0 +1,106 @@
+
+## How does Pulumi relate to `kubectl`?
+
+Most users use Pulumi to manage Kubernetes resources, but still use `kubectl`'s rich commands for
+examining cluster state (_e.g._, `describe`, `explain`, `get`, and so on).
+
+The differences in resource management are:
+
+* Pulumi and `kubectl` both use the official [Kubernetes Go client library][client-go] to talk to do
+  things like load the [kubeconfig file][kubeconfig], authenticate to the cluster, and talk to the
+  API server.
+* The Pulumi CLI presents information-rich status updates, giving a notion of progress as resources
+  come online. `kubectl` does not.
+* Pulumi has a strong notion of a "plan" -- running `pulumi preview` allows you to see what effect
+  a change will have on your resource configuration. `kubectl` does not.
+* Explicit notions of creation, deletion, and replacement. If we must change an immutable field in
+  an API object, `pulumi preview` alerts the user the object must be replaced. `kubectl` does not.
+* Pulumi exposes `pulumi preview --diff`, allowing you to see how a resource has specifically
+  changed. (This will be coming in future releases of `kubectl`.)
+* The primary interface to `kubectl` is YAML. Pulumi exposes a rich, multi-language SDK, and
+  additionally supports raw Kubernetes YAML with the `yaml` namespace in the root of the Kubernetes
+  package.
+* `kubectl` has a set of commands specifically meant to make it easy to understand what's happening
+  in the cluster. Pulumi _only_ does resource management.
+* Pulumi has a notion of resource initialization completion, allowing resources be configured using
+  values from the live object. For example, a deployment might boot a database, then parse the
+  connection string, then put that in a secret, then reference that secret in a `Pod`. As of
+  Kubernetes v1.12, `kubectl` can also wait for resource initialization, though it only supports
+  applying all resource configuration at one time.
+* The ability to deploy the same application multiple times due to autonaming.
+
+## How does Pulumi relate to `helm`?
+
+Helm v2 allows users to easily install a pre-packaged application ("Chart") into a Kubernetes
+cluster. Charts are parameterized by some number of values, which users can fill in to customize
+their application.
+
+* Helm 2 Charts are managed by an in-cluster API server, called Tiller, rather than the official
+  Kubernetes API server. This makes many things (_e.g._, RBAC) harder, because the API server is run
+  with a single `ServiceAccount`, which typically has global read/write access to the cluster.
+
+  Pulumi requires no server-side component. Just like `kubectl`, it uses the official [Kubernetes go
+  client library][client-go] to talk directly to the API server. It's appropriate to drop Pulumi in
+  anywhere you already use `kubectl`.
+* The Pulumi CLI presents information-rich status updates, giving a notion of progress as resources
```

Comment:
Many of these points seem to be about `kubectl` but this section is nominally about `helm`.